src/completers/git.zsh

```diff
@@ -129,8 +129,9 @@ _fzf_complete_git() {
                 ;;
 
             *)
-                if _fzf_complete_git_parse_argument 1 "${arguments%% -- *}" "${(F)git_options_argument_required}" > /dev/null; then
-                    _fzf_complete_git-ls-files '' '--multi' $@
+                local treeish
+                if treeish=$(_fzf_complete_git_parse_argument 1 "${arguments%% -- *}" "${(F)git_options_argument_required}") || [[ -n $treeish ]]; then
```

Comment:
Why does it have to assign to a variable?

Reply:
Because it is to use in `git ls-tree`.

https://github.com/chitoku-k/fzf-zsh-completions/blob/044cda59364c26a63fb7bcb4d13945854463bfd0/src/completers/git.zsh#L589

Reply:
I smply mised that as I searched only for `\$treeish` 🤣